src/app/sections/why-section.tsx

```diff
@@ -4,37 +4,39 @@ import Link from "next/link";
 
 export default function WhySection() {
   return (
-    <section className="flex flex-col items-center justify-center py-16 px-8 md:p-16 bg-white">
-      <h2 className="text-4xl font-medium text-center">{why.headline}</h2>
-      <div className="flex flex-col xl:flex-row justify-around mt-8">
-        {why.items.map((item, index) => (
-          <div key={index} className="flex flex-col items-center p-8">
-            <div className="relative size-76 max-w-[250px] md:max-w-none">
-              <Image
-                className="size-full"
-                src={item.image}
-                alt={`${item.headline} image`}
-                fill
-              />
-            </div>
+    <section className="bg-white">
+      <div className="flex flex-col items-center justify-center py-16 px-8 md:p-16 max-w-7xl m-auto">
+        <h2 className="text-4xl font-medium text-center">{why.headline}</h2>
+        <div className="flex flex-col xl:flex-row justify-around mt-8">
```

Comment:
use justify-between instead of justify-around, the first and last items should touch the container edges

```suggestion
        <div className="flex flex-col xl:flex-row justify-between mt-8">
```